modular_citadel/code/modules/arousal/organs/penis.dm

```diff
@@ -18,23 +18,56 @@
 	var/knot_girth_ratio 	= KNOT_GIRTH_RATIO_DEF
 	var/list/dickflags 		= list()
 	var/list/knotted_types 	= list("knotted", "barbed, knotted")
+	var/prev_length			= 6 //really should be renamed to prev_length
+
+/obj/item/organ/genital/penis/Initialize()
+	. = ..()
+	/* I hate genitals.*/
```

Comment:
yeah i do too after the latest string of erp/meme prs.

Reply:
I'm sorry.